check_tv.py

```
@@ -425,12 +425,12 @@ def edit_live_stream(video_id, new_title, new_description):  # Function to edit
        break  # Break loop
     except Exception as e:  # Handle exceptions
      if hitryagain == 3:  # Check if retry limit is reached
-      logging.info(f"Error and stoping because of error that can't fix")  # Log error
+      logger.info(f"Error and stoping because of error that can't fix")  # Log error
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)